judge/models/choices.py

```diff
@@ -21,6 +21,7 @@ def make_timezones():
 del make_timezones
 
 ACE_THEMES = (
+    ('auto', _('Follow theme default')),
```

Comment:
Note: this isn't sorted so that it's shown first in the dropdown in edit profile.